bucket/cupscale.json

```diff
@@ -0,0 +1,33 @@
+{
+    "version": "1.38.0f1",
+    "description": "Image Upscaler GUI",
+    "homepage": "https://github.com/n00mkrad/cupscale",
+    "license": "MIT",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/n00mkrad/cupscale/releases/download/1.38/Cupscale.1.38.0f1.zip",
+            "hash": "f70f7a84c5984612776ee8d40635a8bae283650049d478c8c49d059ade12b30a"
+        }
+    },
+    "extract_dir": "Cupscale 1.38.0f1",
+    "bin": "Cupscale.exe",
+    "shortcuts": [
+        [
+            "Cupscale.exe",
+            "Cupscale"
+        ]
+    ],
+    "persist": "CupscaleData\\models",
+    "checkver": {
+        "github": "https://github.com/n00mkrad/cupscale",
+        "regex": "/Cupscale\\.([\\w.]+)\\.zip"
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/n00mkrad/cupscale/releases/download/$majorVersion.$minorVersion/Cupscale.$version.zip"
+            }
+        },
+        "extract_dir": "Cupscale $version"

```

Comment:
Usually extract_dir is present within the architecture block itself

Check https://github.com/ScoopInstaller/Extras/blob/master/bucket/staxrip.json for reference

Reply:
This isn't really a requirement. (the global extract_dir is outside the arch block too, so this is fine)

Reply:
Usually Scoop manifests keep both extract_dir and URL within the same blocks so we could do it for standardization.

Reply:
If something is arch-specific, then only it should go inside the arch block. This is the standard to be followed. Otherwise it's common for both archs, so it should be outside.

Reply:
But there is a single arch in this case.

Sane as Staxrip and a lot of other applications

Reply:
Staxrip has the string `x64` in its extract_dir, so it's inside arch block.